src/main/kotlin/com/chattriggers/ctjs/minecraft/wrappers/entity/Entity.kt

```diff
@@ -392,25 +375,26 @@ open class Entity(val entity: net.minecraft.entity.Entity) {
         //#endif
     }
 
-    // TODO(CONVERT): Introduce a CT enum for this
-    //#if MC<=11202
-    fun getDimension() = entity.dimension
-    //#endif
-
-    fun setPosition(x: Double, y: Double, z: Double) = apply {
+    fun getNBT(): NBTTagCompound {
+        val compound = net.minecraft.nbt.NBTTagCompound()
         //#if MC<=11202
-        entity.setPosition(x, y, z)
-        //#else
-        //$$ entity.setPos(x, y, z)
+        entity.writeToNBT(compound)
+        //#elseif MC>=11701
+        //$$ entity.saveWithoutId(compound)
         //#endif
+        return NBTTagCompound(compound)
     }
 
-    fun setAngles(yaw: Float, pitch: Float) = apply {
+    fun getDimension(): World.Dimension {
         //#if MC<=11202
-        entity.setAngles(yaw, pitch)
-        //#else
-        //$$ entity.xRot = yaw
-        //$$ entity.yRot = pitch
+        return World.Dimension.values()[entity.dimension + 1]
```

Comment:
Is this guaranteed to be safe? For example, on the title screen. If not, it might be better to do `.getOrElse(entity.dimension + 1, World.Dimension.Overworld)`.